CMakeLists.txt

```diff
@@ -27,6 +27,7 @@ option(ENABLE_SMESH "Enable SMESH" OFF)
 option(ENABLE_NETGEN "Enable Netgen" OFF)
 option(ENABLE_WARNINGS "Disable warning output." OFF)
 option(ENABLE_FORCE "Force build even if unresolved externals are found" OFF)
+option(TEST_BUILD "Build a subset of modules for testing." OFF)
```

Comment:
Do you find the option for test builds useful? I think I did away with it and the associated code in the azure_support branch, but if there is a use for it it's ok to leave for now. this can be decided later, anyway

Reply:
Yes it was helpful.... mainly to sort out weird issues with conda packaging (to avoid waiting the full 1-2 hrs) for a full build.